website/docs/rbs-support.md

```diff
@@ -63,49 +63,12 @@ an alternative annotation syntax for Ruby. The headline features of RBS:
 
 - It's the type annotation syntax blessed by the Ruby core team, and Matz in
   particular (the lead of the Ruby project).
-- RBS supports certain enticing features, like method overloading and duck
-  types.
 - The syntax is terse.
 - The Ruby distribution bundles a gem for parsing RBS annotations.
 
 However, there are a number of problems which mean that RBS syntax has a
 second-class position in Sorbet:
 
-### RBS syntax does not match the semantics of Sorbet
-
-Sorbet's type annotation syntax evolved differently from the evolution of RBS
-syntax. Unfortunately, this difference is not syntax-deep: it affects the
-semantics of the types too.
-
-Sure, there are similarities: both syntaxes offer a way to express union types,
-for example. But there are also semantic differences, and RBS syntax reflects
-these differences. Things which are possible to express in RBS syntax have no
-analogue in Sorbet and vice versa. Some examples:
-
-- RBS supports duck typing via interfaces (different from Sorbet's
-  [interfaces](abstract.md)), but
-  [Sorbet does not support duck typing](faq.md#can-i-use-sorbet-for-duck-typed-code),
-  by design.
-- Sorbet treats Ruby's `::Class` as a generic class, allowing sophisticated
-  class-level metaprogramming (abstracting over a class's attached class). In
-  turn, all class singleton classes are generic
-  ([`T.class_of(...)[...]`](class-of.md#tclass_of-applying-type-arguments-to-a-singleton-class-type)).
-  RBS does not have syntax to represent this.
-- By extension, Sorbet allows singleton classes to declare their own generic
```

Comment:
You can still use Sorbet's generics the same way

Reply:
(see previous reply)